website/settings/defaults.py

```diff
@@ -411,6 +411,7 @@ class CeleryConfig:
     task_med_queue = 'med'
     task_high_queue = 'high'
     task_remote_computing_queue = 'remote'
+    account_status_changes = 'account_status_changes'
```

Comment:
consistent naming might have helped avoid getting confused when using it
```suggestion
    task_account_status_changes_queue = 'account_status_changes'
```